lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -23,6 +26,10 @@ def release_file
         File.write(miq_dir.join("RELEASE"), RELEASE)
       end
 
+      def link_plugin_public_dirs
+        symlink_plugin_paths("public", miq_dir.join("public"))
```

Comment:
@simaishi I noticed the following line of code, which might cause this to be an issue:

https://github.com/ManageIQ/manageiq-rpm_build/blob/a8d37edafeba9d751c53a6acb72de47a46d71bd2/lib/manageiq/rpm_build/generate_tar_files.rb#L22

Can we discuss this?  I think I understand why, but I'm thinking there might be a better way to do it there, so we don't have to enumerate things individually.

Reply:
https://github.com/ManageIQ/manageiq-rpm_build/pull/18 removes those 2 lines. Working on PR feedback, should be ready for another review soon.